views/components/GraphTable/GraphTable.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+} from '@material-ui/core'
+import { formatSiteData, graphTableColumns } from '../../fe-utils/helpers'
+import { graphTableRowDataBySiteUtil } from './helpers'
+
+const GraphTable = ({ graph }) => {
+  if (!graph) return null
+
+  const tableColumns = graphTableColumns(graph.labels)
+  const [totalsSite, ...sites] = graphTableRowDataBySiteUtil(graph.dataBySite)
+
+  return (
+    <Paper>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center">Site</TableCell>
+            {tableColumns.map(({ name }) => (
+              <React.Fragment key={name + 'table-header'}>
```

Comment:
these keys don't need to be globally unique just unique to this map so we can use the names as the key.

```suggestion
              <React.Fragment key={name}>
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/193/commits/98ef1ad36740b7dcf6eae8a2e3f687620ac21cc2